tests/integration_tests/functional/test_cpu_features_aarch64.py

```diff
@@ -108,6 +109,12 @@ def test_cpu_features_with_static_template(
     guest_kv = re.search(r"vmlinux-(\d+\.\d+)", guest_kernel.name).group(1)
     _check_cpu_features_arm(vm, guest_kv, "v1n1")
 
```

Comment:
Add a comment to make it clear this is a 2nd part... i.e. `# Do the same thing for a restored guest`

Reply:
done